iOS/Flows/Home/Members/OrbCollectionViewLayout.swift

```diff
@@ -66,82 +104,66 @@ class OrbCollectionViewLayout: UICollectionViewLayout {
-        var indexInOrbit: Int = 0
-        // Because the first item is reserved for the center, start on index 1
-        var currentOrbitStartIndex: Int = 1
-
-        // Determine which orbit the item is in and the index of that item within its orbit
-        while true {
-            // The starting index of the next orbit.
-            // We add 1 at the end because the first item is in the center, which shifts everything up 1
-            let nextOrbitStartIndex = (pow(2, orbitIndex + 1) - 1) * self.firstOrbitItemCount + 1
-
-            // Check to see if the index path is in the current orbit
-            if indexPath.row < nextOrbitStartIndex {
-                // If so calculate its index within the current orbit.
-                indexInOrbit = indexPath.row - currentOrbitStartIndex
-                break
-            }
+    // MARK: - Scroll Behavior
+
+    override func targetContentOffset(forProposedContentOffset proposedContentOffset: CGPoint,
+                                      withScrollingVelocity velocity: CGPoint) -> CGPoint {
 
-            orbitIndex += 1
-            currentOrbitStartIndex = nextOrbitStartIndex
+        guard let collectionView = self.collectionView else {
+            return super.targetContentOffset(forProposedContentOffset: proposedContentOffset,
+                                             withScrollingVelocity: velocity)
         }
 
-        // The distance between the previous orbits center and the following orbits circumference
-        let radius = (self.itemSize + self.interimSpace) * CGFloat(orbitIndex + 1)
-        let totalInGivenOrbit = pow(2, orbitIndex) * self.firstOrbitItemCount
-        // The angle between each item in the current orbit
-        let angle = CGFloat(2 * Double.pi) / CGFloat(totalInGivenOrbit)
-        let itemAngle = angle * CGFloat(indexInOrbit) // The actual angle of the item
-        let itemX = (radius * cos(itemAngle)) + center.x
-        let itemY = (radius * sin(itemAngle)) + center.y
+        // Find the item position that is closest to the proposed offset.
+        let proposedCenter = CGPoint(x: proposedContentOffset.x + collectionView.halfWidth,
+                                     y: proposedContentOffset.y + collectionView.halfHeight)
 
-        return CGPoint(x: itemX, y: itemY) // Used to set the center point of the current item
+        let closestPosition = self.itemPositions.min { (position1, position2) in
+            return proposedCenter.distanceTo(position1) < proposedCenter.distanceTo(position2)
+        }
+
+        guard let closestPosition = closestPosition else {
+            return super.targetContentOffset(forProposedContentOffset: proposedContentOffset,
+                                             withScrollingVelocity: velocity)
+        }
+
+        return CGPoint(x: closestPosition.x - collectionView.halfWidth,
+                       y: closestPosition.y - collectionView.halfHeight)
```

Comment:
This is cell centering logic. It always centers on cell, but it kind of feels weird to be honest.